src/generator/Generator.py

```diff
@@ -71,14 +75,14 @@ def latent_space_mean(self) -> np.ndarray:
         pass
 
     @abc.abstractmethod
-    def generate(self, latent_codes: list[np.ndarray]) -> list[str]:
+    def generate(self, latent_codes: np.ndarray) -> list[str]:
         """
         Should generate images for the specified latent codes. This will most likely
         involve the launch of a separate agent process using the EnvironmentManager,
         allowing external generators to be launched in appropriate conda environments.
 
         Args:
-            latent_codes (list[np.ndarray]): The latent codes to generate images for.
+            latent_codes (np.ndarray): The latent codes to generate images for.
```

Comment:
Add "Each entry on the first axis is interpreted as a different latent code." or similar